packages/config/config/devices/0x001d/dzpd3.json

```diff
@@ -85,15 +98,15 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED Off",
+					"label": "Off",
```

Comment:
Please make this Always off for consistency to what we have elsewhere.

Reply:
Done.